### PR TITLE
Docker for mina-archive-maintenance

### PR DESCRIPTION
**Pull request by dkijania**
Added docker for mina-archive-maintenance debian to our CI

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Review by emberian — APPROVED** [review]

**Review by emberian — APPROVED** [review]

**Comment by emberian** [comment]
!ci-nightly-me

**Comment by deepthiskumar** [comment]
!ci-nightly-me

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!ci-nightly-me

**Comment by deepthiskumar** [comment]
!ci-nightly-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

File: scripts/archive/build-release-archives.sh [hunk 1]
```diff
@@ -116,6 +116,7 @@ cat "${BUILD_DIR}/DEBIAN/control"
 
 echo "------------------------------------------------------------"
 # Binaries
+rm -rf "${BUILDDIR}/usr/local/bin"
```
Comment (emberian) [inline scripts/archive/build-release-archives.sh:119]: this is scary to me, i hope it is needed :o

Comment (emberian) [reply scripts/archive/build-release-archives.sh:119]: i see now.

**Pull request merged**